requests_ntlm/requests_ntlm.py

```diff
@@ -31,9 +34,11 @@ def __init__(self, username, password, session=None):
                 self.domain = '.'
 
         self.domain = self.domain.upper()
-
         self.password = password
 
+        # Used by other python libraries to sign/seal messages with NTLM methods
+        self.session_security = None
```

Comment:
Have a look at https://github.com/diyan/pywinrm/issues/127, it is a very rudimentary example of how a library that uses requests-ntlm to authenticate but then require the session_security to encrypt/decrypt the messages after authentication.